index.html

```diff
@@ -6,6 +6,14 @@
     <title>Les 3 - Opdracht CSS Grid</title>
 </head>
 <body>
+<div>
+    <section class="item item-a">A</section>
```

Comment:
elke div zou en id hebben om een naam te geven op de grid-area. Jij hebt  alles gedaan met 2 classes te geven per div. In ieder geval jouw manier is ook goed aangezien de resultaat.